salt/states/boto3_sns.py

```diff
@@ -311,11 +378,11 @@ def topic_absent(name, unsubscribe=False, region=None, key=None, keyid=None, pro
 
 def _json_objs_equal(left, right):
     left = __utils__['boto3.ordered'](
-        salt.utils.json.loads(left)
```

Comment:
Same question here.

Reply:
Ah, because of this at the top :)
```
import salt.utils.json as json
```

Reply:
It seemed a cleaner style to do it that way, but if it's counter to the norm I can revert it of course :)